src/main/java/reactor/core/publisher/LambdaFirstSubscriber.java

```diff
@@ -55,20 +56,29 @@
 	 * @param errorConsumer A {@link Consumer} called onError
 	 * @param completeConsumer A {@link Runnable} called onComplete with the actual
 	 * context if any
+	 * @param subscriptionConsumer A {@link Consumer} called with the
+	 * {@link Subscription}, or null to ignore
 	 */
 	public LambdaFirstSubscriber(Consumer<? super T> consumer,
 			Consumer<? super Throwable> errorConsumer,
-			Runnable completeConsumer) {
+			Runnable completeConsumer,
+			Consumer<? super Subscription> subscriptionConsumer) {
 		this.consumer = consumer;
 		this.errorConsumer = errorConsumer;
 		this.completeConsumer = completeConsumer;
+		this.subscriptionConsumer = subscriptionConsumer;
 	}
 
 	@Override
 	public final void onSubscribe(Subscription s) {
 		if (Operators.validate(subscription, s)) {
 			this.subscription = s;
 			try {
+				//note that unlike in RxJava 2.0.0 an error on accept doesn't trigger
```

Comment:
Reminds me when I reported a bug against JDK 9 Flow implementation and instead of fixing it, they added a javadoc comment of "btw this doesn't actually work".